src/app/model/Mission.ts

```diff
@@ -138,6 +138,36 @@ const fsIncomplete = (orgId: string) => ({
   storeAs: "incompleteMissions",
 });
 
+const fsAvailableUserMissions = (orgId: string, userId: string) => ({
```

Comment:
Can we just make this all one call and then filter it on the frontend?

I know this is how the org dashboard does it, but that should probably get refactored too.

This way we're cutting down how many queries (reads) are getting made.

Reply:
Can you add an issue to do that refactor later? Just being the usual PITA PO and saying hey we gotta get this out by Monday for reasl though. Promise we'll have a refactor party afterwards though!

Reply:
yeah, lets do that later.